scripts/trigger-mobile-metrics.py

```diff
@@ -0,0 +1,61 @@
+#!/usr/bin/env python3
+
+# Implements https://circleci.com/docs/2.0/api-job-trigger/
+
+import os
+import json
+import requests
+import sys
+
+def TriggerPipeline(token, commit, job, run_benchmark):
+    url = "https://circleci.com/api/v1.1/project/github/mapbox/mobile-metrics/tree/km-refactor-navigation-trigger"
```

Comment:
```suggestion
    url = "https://circleci.com/api/v1.1/project/github/mapbox/mobile-metrics/tree/master"
```

The mobile metrics pull request needs to be merged in order for this to work